zzz_modular_syzygy/code/game/jobs/SZdepartment.dm

```diff
@@ -0,0 +1,26 @@
+/datum/department/guild
+	..()
```

Comment:
All child objects of datums inherit from their parents, so putting a callback here is unnecessary. Furthermore, the ..() proc only functions within procs anyways.

Reply:
Done